_jekyll/_layouts/default.html

```diff
@@ -12,7 +12,9 @@
         {% endif %}
     </head>
 
-    <body style="margin: 1rem;">
-{{ content }}
+    <body>
+      <div style="margin: 0 1rem 1rem 1rem;">

```

Comment:
This mark up will show up in the example code snippet as `example-js` displays the contents of the body.